content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,81 @@
+
+For a demonstration of wrinkle mask blending, see a [YouTube video](https://www.youtube.com/watch?v=ebr8t_mz8p8&t=3s) by Deadhaus Sonata. This video shows wrinkle mask blending with O3DE in action. 
+
+The skin shader supports up to 4 wrinkle layers, where a wrinkle layer can be a base color or a normal. A wrinkle mask is an RGBA `.tif` file, and each color channel corresponds to the weight of one of the wrinkle layers. So for a material with only one wrinkle layer, all of the masks will only be red (with `0`, or black, in the green, blue, and alpha channels). If a material uses two wrinkle layers, such as one for vertical wrinkles and one for horizontal wrinkles, then each wrinkle mask will be red, green, or a combination of the two.
+
+{{< important >}}
+This workflow represents a minimum viable product for animating wrinkle masks. There are a number of aspects that could use feedback on how to improve, such as how the normal maps and blend masks are blended, and the potential need for corrective blend masks. If you are using this feature, please reach out to [sig-graphics-audio on Discord](https://discord.com/channels/805939474655346758/816043793576886273) with your suggestions!
+{{< /important >}}
+
+## Setting up wrinkle masks in O3DE
+
+To set up wrinkle masks for your morph targets, you must add the wrinkle masks to the correct location in your project folder and use the appropriate file names. 
+
+1. In the folder that `<mycharacter>.fbx` is in, create a `<mycharacter>_wrinklemasks` folder. Using this exact naming convention, the Asset Processor can associate any wrinkle masks in the wrinkle mask folder with morph targets from the `.fbx`.
+
+    ![Folder Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-folder-naming-convention.png)
+
+1. In the `<mycharacter>_wrinklemasks` folder, add one blend mask for each morph target that activates the wrinkle layers. Consider these additional rules: 
+
+    - If the morph target doesn't activate any wrinkle layers, it doesn't need a mask. 
+
+    - Morph targets cannot share the same masks. So if two morph targets intend to use the same mask, you must duplicate the mask texture instead.
+
+1. Ensure that the blend masks files have the correct  name, following the wrinkle mask naming convention. The name of each mask must match the name of the morph target in the corresponding `.fbx`. 
+
+     In the following images, you can see that the wrinkle mask file names match the morph target names (displayed as shape keys in Blender) and end with `_wrinklemask.tif`.
+
+    ![Wrinkle Mask File Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-file-naming-convention.png)
+    ![Shape Key Names in Blender](/images/atom-guide/materials/skin/shape-key-names-in-blender.png)
+    {{< caution >}}
+If you rename the morph target in the `.fbx`, you must rename the wrinkle mask. 
+    {{< /caution >}}
+
+
```

Comment:
```suggestion
## Authoring wrinkle masks
```

Reply:
Made a Authoring Wrinkle Masks section at the top.